### DISCUSSION TITLE
How to Scan Internet for Failures

### DISCUSSION THREAD

**Discussion by puentejo** [community/community#131984] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way I can scan my internet for failures? I need a way into I can scan my network for failures to report to my internet provider.
Thanks You .

**Comment by ghost** [comment]
Hi , you can use various network monitoring sites that will help you find any outages if you have them. They mainly show network performance, failures and errors.

**Comment by puentejo** [comment]
is there a program here on Github for scanning network for dummies? Im still learning.

**Reply by ghost** [reply]
I searched and found this https://github.com/s0md3v/Smap if you want you can have a look

**Comment by Swastik0710** [comment]


**Reply by ebndev** [reply]
Hi @Swastik0710, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟
